tests/login.test.js

```diff
@@ -0,0 +1,88 @@
+describe('DigitalBank', () => {
+
+    //example test
+    it('Should enter the site', async () => {
+      browser.url('login');
+    });
+
+    it('Should log in to the demo account', async () => {
+      browser.url('login');
+      
+      let username = await $('[id="username"]');
+      await username.setValue('jsmith@demo.io');
+
+       let password = await $('[id="password"]');
+      await password.setValue('Demo123!');
+
+      let loginButton = $('[id="submit"]');
+      await loginButton.click()
+      await browser.pause(5000);
+
+    });
+
+    it('Should log in to the demo account using enter key', async () => {
+      browser.url('login');
+      
+      let username = await $('[id="username"]');
+      await username.setValue('jsmith@demo.io');
+
+       let password = await $('[id="password"]');
+      await password.setValue('Demo123!');
+
+      let loginButton = $('[id="submit"]');
+      await loginButton.keys('Enter');
```

Comment:
Same comment regarding using an **expect** clause at the end of the test to explicitly assert an expected condition